### PR TITLE
Add wallaby.js configuration file

### PR DESCRIPTION
**Pull request by hasparus**
### What's changed

- Rename config.test.ts to config.testing.ts
- Add WallabyJS configuration file

### Relevant links

- https://wallabyjs.com/

### CODE REVIEW

File: wallaby.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config();
+
+module.exports = function (_wallaby) {
+  const packageNames = ["backend", "encoding", "crypto", "frontend"];
+  const packages =
+    packageNames.length === 1 ? packageNames[0] : `{${packageNames.join(",")}}`;
```
Comment (sz-piotr) [inline wallaby.js:7]:
```suggestion
    packageNames.join(",");
```

File: wallaby.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const directories = `./packages/${packages}/{src,test}/**/`;
+  const sourceFiles = directories + "*.{ts,mjs,cjs,js,json,env}";
```
Comment (sz-piotr) [inline wallaby.js:9]:
```suggestion
  const sourceFiles = directories + "*.{ts,tsx,mjs,cjs,js,jsx,json,env}";
```

File: wallaby.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  const testFiles = directories + "*.test.ts";
```
Comment (sz-piotr) [inline wallaby.js:10]:
```suggestion
  const testFiles = directories + "*.test.{ts,tsx}";
```

File: wallaby.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config();
```
Comment (sz-piotr) [inline wallaby.js:2]: Why is this necessary?

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: wallaby.js (hunk 1 above)
Comment (hasparus) [reply wallaby.js:7]: Your suggestion drops `{` and `}`.

If there is more than one directory name in the array we want `"{one,two}"`, otherwise we just want `"one", because `"{one}"` is illegal.

File: wallaby.js (hunk 4 above)
Comment (hasparus) [reply wallaby.js:2]: The file itself or `require("dotenv").config()`?

Comment (hasparus) [reply wallaby.js:2]: If you're asking about `dotenv`, I wanted to let Wallaby know about `process.env.TEST_DB_URL`.

Comment (sz-piotr) [reply wallaby.js:2]: The require call

File: wallaby.js (hunk 1 above)
Comment (sz-piotr) [reply wallaby.js:7]: Oh! I didn't notice the brackets. Then the code is correct
